ratis-server-api/src/main/java/org/apache/ratis/server/RaftServerConfigKeys.java

```diff
@@ -119,6 +119,17 @@ static SizeInBytes byteLimit(RaftProperties properties) {
     static void setByteLimit(RaftProperties properties, SizeInBytes byteLimit) {
       setSizeInBytes(properties::set, BYTE_LIMIT_KEY, byteLimit, requireMin(1L));
     }
+
+    String FOLLOWER_MAX_GAP_RATIO_KEY = PREFIX + ".follower-max-gap-ratio";
+    float FOLLOWER_MAX_GAP_RATIO_DEFAULT = 1;
```

Comment:
Let's use double instead of float.  We should avoid using float, especially in 64-bit computers.

Reply:
Sure.  Except for that double is more precise than float,  any major drawbacks of using float?

Reply:
Precision is the main reason.

Also, double is the default in Java.  For example, the code below does not compile.
```
      float a = 1.2;
```
Using float is easier to make mistakes unawarely.

Unless space is an issue, we should just use double.